File: src/main/java/compare_the_triplets/CompareTheTriplets.java
package compare_the_triplets;

import java.util.Arrays;
import java.util.List;

public class CompareTheTriplets {
    public void compareTheTriplets(){
        int alice_score = 0;
        int bob_score = 0;
        List<Integer> alice = Arrays.asList(17,28,30);
        List<Integer> bob = Arrays.asList(99,16,8);

        for(int i = 0; i<alice.size(); i++){
            if(alice.get(i) > bob.get(i)){
                alice_score++;
            }

            if(bob.get(i) > alice.get(i)){
                bob_score++;
            }

            if(bob.get(i).equals(alice.get(i))){
                //do nothing
            }

        }
        System.out.println(alice_score);
        System.out.println(bob_score);
    }
}

File: src/main/kotlin/compare_the_triplets/compareTheTriplets.kt
package compare_the_triplets

fun compareTheTriplets() {
    var aliceScore = 0
    var bobScore = 0

    val listAlice = listOf(17, 28, 30)
    val listBob = listOf(99, 16, 8)

    for (i in listAlice.indices) {
        if (listAlice[i] > listBob[i]) {
            aliceScore++
        }

        if (listBob[i]>listAlice[i]){
            bobScore++
        }

        when {
            listAlice[i]==listBob[i] -> {
                //do nothing
            }
        }
    }
    println(aliceScore)
    println(bobScore)

}